Cubical/Data/Nat/Coprime.agda

```diff
@@ -0,0 +1,47 @@
+{-# OPTIONS --cubical --safe #-}
+module Cubical.Data.Nat.Coprime where
+
+open import Cubical.Foundations.Prelude
+open import Cubical.Foundations.Function
+
+open import Cubical.Data.NatPlusOne.Base
+
+open import Cubical.Data.Prod hiding (_×_) renaming (_×Σ_ to _×_)
+
+open import Cubical.HITs.PropositionalTruncation as PropTrunc
+
+open import Cubical.Data.Nat.Base
+open import Cubical.Data.Nat.Properties
+open import Cubical.Data.Nat.Divisibility
+open import Cubical.Data.Nat.GCD
+
+areCoprime : ℕ × ℕ₊₁ → Type₀
+areCoprime (m , n) = isGCD m (ℕ₊₁→ℕ n) 1
```

Comment:
Why not define this for ℕ × ℕ ?

Reply:
This is nicer because the relation is symmetric and no casts are needed

Reply:
Done. Note that `toCoprime` does need one of its arguments to be nonzero.